### DISCUSSION TITLE
Take one image as "dimensions template" and use it for resize another images

### DISCUSSION THREAD

**Discussion by stalkerGH** [ImageMagick/ImageMagick#7090] [Help] [answered]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Ubuntu 22.04

### Description

_I asked this question on StackOverflow but in the meantime found this discussion forum so I ask here again._

I'd like to know syntax for such task: I have one file which is "dimensions template" - it has its own WxH. IM should read this dimensions and use them for resizing of other file ONLY.

As for now, I have such command working:

`magick template.jpg image-to-resize.jpg -resize "x%[fx:int(h/2)]" second-image-resized.jpg`

But calling this command outputs two images - first and second taken as input - resized. How to tell 'fx' that it should read dimensions from first input image?

**Comment by snibgo** [comment, marked as answer]
Your command resizes both images, but you don't need to resize the first image. You don't even need to read its pixels. I suggest (bash syntax):
```
magick \
  -ping template.jpg \
  -set option:HH %h \
  +delete \
  +ping image-to-resize.jpg \
  -resize "x%[fx:HH/2]" \
  second-image-resized.jpg
```

**Reply by stalkerGH** [reply]
Great solution, thank you. I will link your answer on StackOverflow, if someone need similar thing to do.
